### DISCUSSION TITLE
Support tsconfig paths without baseUrl

### DISCUSSION THREAD

**Discussion by Janpot** [vercel/next.js#34535] [Ideas]
### Describe the feature you'd like to request

Typescript supports `paths` [without specifying `baseUrl`](https://github.com/microsoft/TypeScript/pull/40101). This is interesting when you want to use some path aliases without allowing bare imports to resolve from the project root.

### Describe the solution you'd like

next.js already supports `paths` with `baseUrl`. It should just follow what typescript does and resolve `paths` from the config location when `baseUrl` is not specified.

I opened a PR to support this behavior: https://github.com/vercel/next.js/pull/34926

### Describe alternatives you've considered

Configure `baseUrl`

**Comment by sisp** [comment]
I agree with the proposal. Any plans to integrate this feature into Next.js?
